sopel/irc/backends.py

```diff
@@ -80,6 +84,21 @@ def __init__(
         self._keyfile: Optional[str] = keyfile
         self._verify_ssl: bool = verify_ssl
         self._ca_certs: Optional[str] = ca_certs
+        self._ssl_ciphers = ":".join(ssl_ciphers)
+
+        self._ssl_minimum_version = ssl_minimum_version
+        if isinstance(ssl_minimum_version, str):
+            self._ssl_minimum_version = getattr(
+                ssl.TLSVersion,
+                ssl_minimum_version,
+                None,
+            )
+        if self._ssl_minimum_version is None:
```

Comment:
I'm not sure it's relevant if the bot doesn't use SSL/TLS:

```suggestion
        if self._use_ssl and self._ssl_minimum_version is None:
```

Reply:
True!